pkgs/applications/altcoins/default.nix

```diff
@@ -1,4 +1,4 @@
-{ callPackage, pkgs }:
+{ callPackage, openssl_1_1_0, boost163, pkgs }:
```

Comment:
Non-critical comment: since the `openssl_1_1_0` and `boost163` args are only used by the `zcash` package why not use the attributes in `pkgs` instead?